src/ja.json

```diff
@@ -0,0 +1,217 @@
+{
+      "welcome-to-rainway" : "Rainwayへようこそ",
+      "no-account-yet" : "まだアカウントを持っていない?",
+      "register-now" : "今すぐ登録",
+      "forgot-password" : "パスワードを忘れた",
+      "register-new-account" : "新しくアカウントを登録",
+      "reset-your-password" : "パスワードをリセットする",
+      "email" : "メールアドレス",
+      "password" : "パスワード",
+      "username" : "ユーザーネーム",
+      "confirm-password" : "パスワードを確認する",
+      "sign-in" : "サインイン",
+      "register" : "登録",
+      "reset-password" : "パスワードリセット ",
+      "save-changes" : "変更を保存",
+      "cancel" : "キャンセル",
+      "submit" : "送信",
+      "download-dashboard" : "ダッシュボードをダウンロード",
+      "support" : "サポート",
+      "finding-servers" : "サーバーを探しています...",
+      "webrtc-connection-failed" : "WebRTCとの接続に失敗しました",
+      "lost-webrtc-connection" : "WebRTCとの接続が失われました.",
+      "connecting" : "接続しています",
+      "no-online-dashboard" : "オンラインになっているダッシュボードが見つかりません",
+      "no-dashboard-message" : "Rainwayクライアントでゲームをストリーミングしてプレイするには、ゲームがインストールされているWindows10を搭載したコンピュータにRainwayダッシュボードをインストール必要があります。これは1度だけのセットアップで、完了には1分もかかりません。",
+      "log-out" : "ログアウト",
+      "or" : "または",
+      "go-back" : "戻る",
+      "outdated-server" : "サーバーのバージョンが古くなっています!",
+      "outdated-server-message" : "このホストで実行されているサーバーのバージョンとは互換性がない原因となった、Rainwayクライアントの大きな変更があります。このコンピュータに接続する前に、Rainwayサーバーを更新してください。",
+      "server-running-here" : "このコンピュータでは既にサーバーが稼働しています!",
+      "server-running-here-message" : "このサーバーをホストしているコンピュータでRaiwayクライアントを使用しようとしています。Rainwayダッシュボードを使用すると、このコンピュータでゲームをローカルに起動できます。Rainwayクライアントでゲームをストリーミングするには、サーバーをホストしているコンピュータ以外のデバイスから接続してください。",
+      "all-games" : "すべてのゲーム",
+      "we-found-games-message" : "システムに{{count}}のゲームがインストールされています",
+      "we-found-games-message_plural" : "システムに{{count}}のゲームがインストールされています",
+      "no-gamepad" : "ゲームパッドなし",
+      "connected-to-how" : "{{name}} ({{how}})に接続済み",
+      "connected-how" : "({{how}})に接続済み",
+      "not-yet-played" : "Rainwayでまだプレイしていません",
+      "minutes-played" : "{{count}}分プレイしました",
+      "minutes-played_plural" : "{{count}}プレイしました",
+      "hours-played" : "{{count}}時間プレイしました",
+      "hours-played_plural" : "{{count}}時間プレイしました",
+      "connected-accounts" : "アカウントに接続",
+      "pending" : "保留中...",
+      "connected" : "接続済み",
+      "not-connected" : "未接続",
+      "download" : "ダウンロード",
+      "all-genres" : "すべてのゲーム",
+      "search" : "検索",
+      "games" : "ゲーム",
+      "settings" : "設定",
+      "account-settings-tab" : "アカウント",
+      "hardware-settings-tab" : "ハードウェア",
+      "advanced-settings-tab" : "詳細設定",
+      "keybindings-settings-tab" : "キー設定",
+      "games-settings-tab" : "ゲーム",
+      "stream-settings-tab" : "システム",
+      "language-settings-tab" : "言語",
+      "about-rainway" : "Rainwayについて",
+      "third-party-licenses" : "サードパーティ ライセンス",
+      "multiple-dashboards-found" : "複数のダッシュボードが見つかりました",
+      "multiple-dashboards-found-message" : "Rainwayダッシュボードをホストする{{count}}デバイスを見つけました。続行するには、接続先を選択してください。",
```

Comment:
Okay, as a Japanese learner, I'm curious 🙂: wouldn't you say 「デバイスを3つ見つけました」 rather than 「3デバイスを見つけました」?

Reply:
That sounds good.
It is sometimes called "1つ". But generally speaking, computers are counted as "1台". However, when it comes to large computers such as server machines, the way of counting changes to "1基".
To be honest, the way of counting varies depending on the item, so even I in Japan can't understand everything: (
Therefore, even if you are not familiar with it, there should be no problem if you understand the meaning.

Rainway is hosted on computers that fall into several categories, but we changed the count to "1台" since many of our users will probably be hosted on desktops.
Even a server can count as "1台". That's fine too.